packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -1924,9 +1924,6 @@
 	"contextual.users.settings.unavailable.button.desktop": "View plans",
 	"contextual.communityNodes.unavailable.description.desktop": "Community nodes feature is unavailable on desktop. Please choose one of our available self-hosting plans.",
 	"contextual.communityNodes.unavailable.button.desktop": "View plans",
-	"contextual.upgradeLinkUrl": "https://n8n.io/pricing/",
-	"contextual.upgradeLinkUrl.cloud": "https://app.n8n.cloud/account/change-plan",
-	"contextual.upgradeLinkUrl.desktop": "https://n8n.io/pricing/?utm_source=n8n-internal&utm_medium=desktop",
```

Comment:
this is still used in SettingsCommunityNodesView.. maybe we can just delete the desktop reference there

Reply:
wth I did a text search

Reply:
All should use the same function in the UI store to redirect to the upgrade path. So will update to use the function and we can get rid of the context.upgradeLinkUrl.*